besu/src/main/java/org/hyperledger/besu/cli/options/MiningOptions.java

```diff
@@ -260,32 +259,6 @@ public void validate(
               + "or specify mining is enabled (--miner-enabled)");
     }
 
-    // Check that block producer options work
-    if (!isMergeEnabled && genesisConfigOptions.isEthHash()) {
-      CommandLineUtils.checkOptionDependencies(
-          logger,
-          commandLine,
-          "--miner-enabled",
-          !isMiningEnabled,
-          asList(
-              "--miner-coinbase",
-              "--min-gas-price",
-              "--min-priority-fee",
-              "--min-block-occupancy-ratio",
-              "--miner-extra-data"));
-
-      // Check that mining options are able to work
-      CommandLineUtils.checkOptionDependencies(
-          logger,
-          commandLine,
-          "--miner-enabled",
-          !isMiningEnabled,
-          asList(
-              "--miner-stratum-enabled",
-              "--Xminer-remote-sealers-limit",
-              "--Xminer-remote-sealers-hashrate-ttl"));
```

Comment:
this isn't just PoW, this is clique and PoA right?  miner has never been the right name, but IIRC PoA algorithms still need to use miner-enabled and this check I believe is still valid for sealer opts